lib/inspec/utils/telemetry/http.rb

```diff
@@ -1,11 +1,16 @@
 # frozen_string_literal: true
 require_relative "base"
 require "faraday" unless defined?(Faraday)
+require "inspec/utils/licensing_config"
 module Inspec
   class Telemetry
     class HTTP < Base
       TELEMETRY_JOBS_PATH = "v1/job"
-      TELEMETRY_URL = ENV["TELEMETRY_URL"] # TODO Set to production telemetry URL before release
+      TELEMETRY_URL = if ChefLicensing::Config.license_server_url&.match?("acceptance")
+                        ENV["TELEMETRY_URL"]
```

Comment:
Could we change this ENV var name to have a CHEF_ prefix, or would that break the world?

Reply:
done